M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -27,9 +27,22 @@ newPackage(
     Authors => {{ Name => "Nathan Grieve", Email => "nathan.m.grieve@gmail.com",
 	    HomePage => "https://sites.google.com/view/nathan-grieve"}},
     AuxiliaryFiles => false,
-    DebuggingMode => false,
+    DebuggingMode => true,
```

Comment:
The builds are failing because `DebuggingMode` should be false (or just removed entirely, since false is the default).